### ISSUE TITLE
how to know the accuracy from this model?

### ISSUE THREAD

**Issue by mega12345678** [remicres/otbtf_tutorials#8]
I have run this model (create_model1.py). But, I still don't know how to know the accuracy. The result is ok. 

otbuser@fd3098e1f203:/home/mega/Documents/CNN$ otbcli_TensorflowModelServe \
> -source1.il Sentinel-2_B4328_10m.tif \
> -source1.rfieldx 16 \
> -source1.rfieldy 16 \
> -source1.placeholder "x" \
> -model.dir "data/results/SavedModel_cnn" \
> -output.names "prediction" \
> -out "classif_model1.tif?&box=4000:4000:1000:1000"
2023-09-01 02:03:52 (INFO) TensorflowModelServe: Default RAM limit for OTB is 256 MB
2023-09-01 02:03:52 (INFO) TensorflowModelServe: GDAL maximum cache size is 186 MB
2023-09-01 02:03:52 (INFO) TensorflowModelServe: OTB will use at most 2 threads
2023-09-01 02:03:52.713603: I tensorflow/cc/saved_model/reader.cc:43] Reading SavedModel from: data/results/SavedModel_cnn
2023-09-01 02:03:52.717680: I tensorflow/cc/saved_model/reader.cc:78] Reading meta graph with tags { serve }
2023-09-01 02:03:52.717747: I tensorflow/cc/saved_model/reader.cc:119] Reading SavedModel debug info (if present) from: data/results/SavedModel_cnn
2023-09-01 02:03:52.717995: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE3 SSE4.1 SSE4.2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-09-01 02:03:52.770161: I tensorflow/cc/saved_model/loader.cc:228] Restoring SavedModel bundle.
2023-09-01 02:03:52.815464: I tensorflow/core/profiler/lib/profiler_session.cc:110] Profiler session initializing.
2023-09-01 02:03:52.815551: I tensorflow/core/profiler/lib/profiler_session.cc:125] Profiler session started.
2023-09-01 02:03:52.817447: I tensorflow/core/profiler/lib/profiler_session.cc:67] Profiler session collecting data.
2023-09-01 02:03:52.817514: W tensorflow/core/profiler/convert/xplane_to_step_stats.cc:74] GPU trace was not collected.
2023-09-01 02:03:52.817712: I tensorflow/core/profiler/lib/profiler_session.cc:143] Profiler session tear down.
2023-09-01 02:03:52.818297: I tensorflow/cc/saved_model/loader.cc:301] SavedModel load for tags { serve }; Status: success: OK. Took 104713 microseconds.
2023-09-01 02:03:52 (INFO): Loading metadata from official product
2023-09-01 02:03:52 (INFO) TensorflowModelServe: Source info :
2023-09-01 02:03:52 (INFO) TensorflowModelServe: Receptive field  : [16, 16]
2023-09-01 02:03:52 (INFO) TensorflowModelServe: Placeholder name : x
2023-09-01 02:03:52 (INFO) TensorflowModelServe: Output spacing ratio: 1
2023-09-01 02:03:52 (INFO) TensorflowModelServe: Output field of expression: [1, 1]
2023-09-01 02:03:52 (INFO) TensorflowModelServe: Force tiling with squared tiles of [16, 16]
2023-09-01 02:03:52 (INFO): Writing user defined region [4000, 4999]x[4000, 5000]
2023-09-01 02:03:52 (INFO): Estimated memory for full processing: 74.0692MB (avail.: 256 MB), optimal image partitioning: 1 blocks
2023-09-01 02:03:52 (INFO): File classif_model1.tif will be written in 1 blocks of 1000x1000 pixels
Writing classif_model1.tif?&box=4000:4000:1000:1000...: 100% [**************************************************] (1m 02s)

Thank you.
